lib/adminui.js

```diff
@@ -45,6 +45,14 @@ function setUfds(req, res, next) {
             next();
         } else if (req.ufdsSlave.connected) {
             req.ufds = req.ufdsSlave;
+
+            assert.string(req.route.name, 'req.route.name');
+            var isAuthRequest = req.route.name === 'authenticate';
+            if (!isAuthRequest && (req.method === 'POST' || req.method === 'PUT' || req.method === 'PATCH')) {
+                req.log.warn('Slave UFDS write attempt is detected.');
+                next(new restify.ServiceUnavailableError('UFDS service is currently not available. Please try later.'));
```

Comment:
Please change the error message to
"Cannot connect to UFDS master. Please verify that the UFDS master service is running and there is no connectivity issue."